ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -434,6 +445,22 @@ public void setFontStyle(@Nullable String fontStyleString) {
     }
   }
 
+  @ReactProp(name = ViewProps.TEXT_DECORATION_LINE)
+  public void setTextDecorationLine(@Nullable String textDecorationLineString) {
+    mIsUnderlineTextDecorationSet = false;
+    mIsLineThroughTextDecorationSet = false;
+    if (textDecorationLineString != null) {
+      for(String textDecorationLineSubString:textDecorationLineString.split(" ")) {
```

Comment:
nit: `for (String textDecorationLineSubString : textDecorationLineString.split(" ")) {`